README.md

```diff
@@ -519,6 +534,38 @@ docker-compose -f examples/test9-compose.yaml up --build --force-recreate
 docker-compose -f examples/test9-compose.yaml exec node ping 10.0.1.2
 ```
 
+### test10: port forwarding
+
+This example demonstrates port forwarding from the conlink container
+to two containers running simple web servers.
+
+Start the test10 compose configuration:
+
+```
+docker-compose -f examples/test10-compose.yaml up --build --force-recreate
+```
+
+Port 80 are published by the conlink container on the host and
```

Comment:
This confused me a bit at first, and I think it should be "Ports 80 and 81 are published ...".  Am I right that there are actually two forwarding/mapping steps here, one using conlink and one using standard Docker port mapping?  (After a few rewrites, I can see how hard this is to describe succinctly...). Perhaps:

> Ports 80 and 81 are published on the host by the conlink container using standard Docker port mapping.
> conlink then forwards from ports 80 and 81 to the first and second replica (respectively) of node1, each
> of which listen internally on port 80.

Just brainstorming, though, so take it or leave it.

Reply:
Yes, there are two steps. And yes, it's hard to described. I adopted your wording and tweaked it slightly. I also re-ordered node1 and node2 so that the simpler (non-replica) case is first. Let me know if the new wording is clearer for that section.